session_0/fizz_buzz.rb

```diff
@@ -36,6 +36,27 @@
 #
 # We will raise an `ArgumentError` exception to let the caller know that
 # their function arguments were incorrect.
-def fizz_buzz(n:, x: 3, y: 5)
-  raise NotImplementedError # TODO
-end
+def fizz_buzz(n:, x: 3, y: 5 )
+  if (n < 0) || (x<= 0) || (y<=0) then
+     raise ArgumentError # TODO
+    end
+   a = []
+   for i in 1..n do
```

Comment:
Do you think map function can be of use here?

https://www.rubyguides.com/2018/10/ruby-map-method/